configuration/locationtagmaps/locationtagmaps-site-cange.csv

```diff
@@ -3,3 +3,4 @@ Cange,,,,,,,,,,,,,,,,,,,TRUE,,,,,,,TRUE,,,,,,,,TRUE,
 Sante Mantal | Cange,,,,,,TRUE,,TRUE,,,,,,,,,TRUE,,,TRUE,,,,,,,TRUE,TRUE,,,,,,,TRUE
 Sante Fanm | Cange,,,,,,TRUE,,TRUE,,,,,TRUE,,,TRUE,TRUE,TRUE,,,,,TRUE,TRUE,,,TRUE,TRUE,,,,,,,TRUE
 Klinik MNT | Cange,,,,,,TRUE,,TRUE,,TRUE,,,,,,TRUE,TRUE,,,,TRUE,,,,TRUE,,TRUE,TRUE,,,,,,,TRUE
+Sal Timoun | Cange,TRUE,TRUE,,,TRUE,,,TRUE,,,,,,,TRUE,TRUE,TRUE,TRUE,,TRUE,,,TRUE,TRUE,TRUE,,TRUE,,,,,TRUE,,,TRUE
```

Comment:
Do we have inpatient functionality at Cange?  If not, probably makes sense to remove the admission and transfer tags (though if the ADT component is turned off at Cange it shouldn't make much of a difference).  Also, I would think that Sal Timoun would be a Check-In location, but it does look like you just copied this from Thomonde Sal Timoun, so maybe not.

Reply:
I actually copied it from La Colline, since LC's locations are a superset of whats at Cange, and removed the one column that doesn't exist here, but otherwise did not make any changes.

Reply:
Works for me